### PR TITLE
Refresh of writing Rust policies.

### LINKED ISSUE #294: Refresh writing policies in Rust

### PR DESCRIPTION
**Pull request by jhkrug**
In the third, 03-define-policy-settings, I am having trouble with the worked Rust examples. I need a little help. Thanks.

When complete and merged will close #294.

### CODE REVIEW

File: docs/writing-policies/rust/03-define-policy-settings.md [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+   Compiling demo v0.1.0 (/home/jhk/projects/suse/tmp/demo)
+error[E0063]: missing field `invalid_names` in initializer of `settings::Settings`
+  \-\-> src/lib.rs:80:23
+   |
+80 |             settings: Settings {},
+   |                       ^^^^^^^^ missing `invalid_names`
+
+Three of those errors. Any suggestions?
+
+-->
```
Comment (jvanz) [inline docs/writing-policies/rust/03-define-policy-settings.md:25]: As far as I can see from the error message, your tests need to be updated after you added the field `invalid_names` in the settings struct. The tests seems to be in the previous version of the code when you do not have the `invalid_names` field yet. You need something like this:

```rust
        let tc = Testcase {
            name: String::from("Valid name"),
            fixture_file: String::from(request_file),
            expected_validation_result: true,
            settings: Settings {invalid_names: HashSet::from(["invalid name"])},
        };
```

Comment (jhkrug) [reply docs/writing-policies/rust/03-define-policy-settings.md]: Thanks @jvanz. All good.

File: docs/writing-policies/rust/01-intro-rust.md [hunk 2]
```diff
@@ -1,47 +1,38 @@
...
-title: ""
+title: "Rust"
+description: An introduction to writing Kubewarden policies with Rust.
+keywords: [kubewarden, kubernetes, writing policies, introduction]
+doc-type: [tutorial]
+doc-topic: [writing-policies, rust, introduction]
 ---
 
-# Rust
+[Rust](https://www.rust-lang.org/) is a mature programming language that can generate WebAssembly modules.
```
Comment (viccuad) [inline docs/writing-policies/rust/01-intro-rust.md:9]:
```suggestion
[Rust](https://www.rust-lang.org/) is the most mature programming language that can generate WebAssembly modules.
```

I would keep the pointer here, it's still true, and seems that will be for the foreseeable. It also explains why we favour rust for our policies.

**Review by viccuad — APPROVED** [review]
LGTM, thanks! Just a minor quibble

File: docs/writing-policies/rust/01-intro-rust.md (hunk 2 above)
Comment (flavio) [reply docs/writing-policies/rust/01-intro-rust.md:9]: I agree with what @viccuad said

Comment (jhkrug) [reply docs/writing-policies/rust/01-intro-rust.md]: Thanks, and fixed.

**Comment by jhkrug** [comment]
Hi all, it's still draft, I want another read through, but I've marked it as ready for review, to get some eyes on it. Thanks. Especially @jvanz for patiently guiding me through the rusty bits.

**Review by jvanz — APPROVED** [review]

**Pull request merged**